lib/openshift/namespace.rb

```diff
@@ -0,0 +1,5 @@
+module BushSlicer
+  class Namespace < ProjectResource
+    RESOURCE = "namespaces"
```

Comment:
I don't think `namespace` is a namespaces resource. Must be `ClusterResource` as far as I can tell.

Reply:
ok, i'll change that.